terminal_test.go

```diff
@@ -305,13 +306,19 @@ var rendererTestCases = []struct {
 		"hello \x1b_bk;t=123\x07world",
 		`<?bk t="123"?>hello world`,
 	}, {
-		`prefixes lines with the first timestamp seen`,
+		`prefixes lines with the last timestamp seen`,
 		"hello\x1b_bk;t=123\x07 world\x1b_bk;t=456\x07!",
-		`<?bk t="123"?>hello world!`,
+		`<?bk t="456"?>hello world!`,
 	}, {
 		`handles timestamps across multiple lines`,
-		"hello\x1b_bk;t=123\x07 world\x1b_bk;t=234\x07!\nanother\x1b_bk;t=345\x07 line\x1b_bk;t=456\x07!",
-		`<?bk t="123"?>hello world!` + "\n" + `<?bk t="345"?>another line!`,
+		strings.Join([]string{
+			"hello\x1b_bk;t=123\x07 world\x1b_bk;t=234\x07!",
+			"another\x1b_bk;t=345\x07 line\x1b_bk;t=456\x07!",
+		}, "\n"),
+		strings.Join([]string{
+			`<?bk t="234"?>hello world!`,
+			`<?bk t="456"?>another line!`,
+		}, "\n"),
```

Comment:
It's just a test, so it's moot, but I wonder if the compiler in lines this? At least with the alternative using `+`, you have this garantee: https://go.dev/ref/spec#Constant_expressions

Reply:
I wasn't 100% sure, so I compiled a test binary and went to crack it open with Ghidra, except I don't have it installed, and it relies on the JDK. Which reminded me that Java (at some point) compiled repeated string concatenations with + into a `StringBuilder` under the hood.

Anyway, I expect the answer is no. Certainly it doesn't create a single string constant.